package/gluon-radvd/files/etc/init.d/gluon-radvd

```diff
@@ -7,7 +7,7 @@ SERVICE_DAEMONIZE=1
 
 
 start() {
-	service_start /usr/sbin/uradvd -i br-client -p $(lua -e 'print(require("gluon.site_config").prefix6)')
+	service_start /usr/sbin/uradvd $(/lib/gluon/radvd/arguments)
```

Comment:
I think the start function should check if */lib/gluon/radvd/arguments* exists and is executable before trying to use it.